### PR TITLE
updates package.json with the proper makara-amdify

### PR DESCRIPTION
**Pull request by gabrielcsapo**
- tests for functionality

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -36,12 +36,22 @@ describe('index', function () {
...
+
                 .expect(/Hello, /)
-            
+
             .end(function (err, res) {
                 done(err);
             });
     });
 
+    it('should work with bcp47 fallback case', function(done) {
```
Comment (grawk) [inline test/index.js]: it's a non-bcp47 compliant case, as opposed to a "fallback"

**Comment by grawk** [comment]
looks good except for the test name.

**Comment by gabrielcsapo** [comment]
@grawk fixed 😄

**Comment by gabrielcsapo** [comment]
thank you @grawk!

**Pull request merged**